collection.go

```diff
@@ -57,11 +59,16 @@ func (c *Collection) SetTableName(tableName string) *Collection {
 	return c
 }
 
+func (c *Collection) SetSchemaName(schemaName string) *Collection {
+	c.schemaName = schemaName
+	return c
+}
+
 func (c *Collection) schemaTableName() (string, string) {
 	if ind := strings.IndexByte(c.tableName, '.'); ind >= 0 {
```

Comment:
You would specify `public.users` and it would see the `.` here and properly return a schema name. There is no need to actually specify the schema anywhere else.